docs/_extra/api-reference/hypothesis-v1.yaml

```diff
@@ -473,9 +473,11 @@ paths:
           in: query
           example: '8JmD3iz1'
           description: |
-            Limit the results to annotations made in the specified group (by group ID).
+            Limit the results to annotations made in the specified group.
```

Comment:
Ok so the parameter is already named `group` which works for both types now :+1: